packaging/DISTRIBUTIONS.md

```diff
@@ -0,0 +1,38 @@
+# Netdata distribution support matrix
+![](https://raw.githubusercontent.com/netdata/netdata/master/web/gui/images/packaging-beta-tag.svg?sanitize=true)
+
+In the following table we provide the list of officially supported operating systems.
+We detail the Distributions, flavours and the level of support Netdata is currently capable to provide.
+
+Kindly note that the following table is a work in progress. We have concluded on the list of distributions
+that we currently supporting and we are working on documenting our current state so that our users
+have complete visibility over the range of support
+
+Distribution | Family| Code health | Installer support | Kickstart support | Binary packaging support | Integrity testing (CI) | Functionality testing (CI) | Community support
+:------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :--------------------
+14.04.6 LTS (Trusty Tahr) | Ubuntu |  |  |  |  |  |  | 
+16.04.6 LTS (Xenial Xerus) | Ubuntu |  |  |  |  |  |  | 
+18.04.2 LTS (Bionic Beaver) | Ubuntu |  |  |  |  |  |  | 
+19.04 (Disco Dingo) Latest | Ubuntu |  |  |  |  |  |  | 
+Debian 7 (Wheezy) | Debian |  |  |  |  |  |  | 
```

Comment:
debian 7 is rip

Reply:
Sure, but [this](https://wiki.debian.org/LTS/Extended) mentions extended support starting with 7

Reply:
In general the idea is that we follow LTS from the official distributions, but we ought to support our users that still remain on versions that recently got outdated anyway.

Reply:
This ELTS thing for debian 7 is not official tho. It is commercial service

> This service provides security support for Debian 7 Wheezy, albeit only on the subset of packages used by the customers of this service.